src/taipy/core/notification/event.py

```diff
@@ -32,7 +32,7 @@ class EventEntityType(_ReprEnum):
     JOB = 6
 
 
-_NO_ATTRIBUTE_NAME_OPERATIONS = set([EventOperation.CREATION, EventOperation.DELETION, EventOperation.SUBMISSION])
+_NO_ATTRIBUTE_NAME_OPERATIONS = set([EventOperation.CREATION, EventOperation.SUBMISSION])
```

Comment:
DELETION is used to model the deletion of an entity. not the deletion of an item of the `properties` attribute.

```suggestion
_NO_ATTRIBUTE_NAME_OPERATIONS = set([EventOperation.CREATION, EventOperation.DELETION, EventOperation.SUBMISSION])
```